src/lambda/commands/uploadToAwsCommand.ts

```diff
@@ -0,0 +1,59 @@
+/*!
```

Comment:
Should live in `src/shared/` not `.../lambda/`. This file is not lambda-specific.